src/Mod/Arch/ArchWall.py

```diff
@@ -996,6 +998,7 @@ def onBeforeChange(self,obj,prop):
 
         if prop == "Length":
             self.oldLength = obj.Length.Value
+        ArchComponent.Component.onBeforeChange(self,obj,prop)
```

Comment:
can you add whitespace after each comma? (pep8 compatible :wink:)

Reply:
Arch code is not pep8 compliant. Doing this for only this line would not make sense.

Reply:
I suggest running flake8 and black to make the whole file beautiful. Then it will make sense ;)

Reply:
What I am more concerned about is the use of `ArchComponent.Component` instead of `super`. Why is that?

Regarding pep8:
I do not know if pep8 compliance is a target for the Arch WB. Maybe @yorikvanhavre can comment.

Reply:
As far as I know, there is no specific compliance other than the normal norm, that code has to be of the same style. Then pep8 is just a norm for doing the same code style. But let's here from others.

Reply:
Arch WB should become somewhat PEP8 compliant (some exceptions may apply). Arch WB references `Draft/coding_conventions.md` which in turn references PEP8.

Reply:
The status of the Draft `coding_conventions.md` file is questionable. I see new Draft code that does not comply. But please let's stop talking about pep8. In the end it is a tool not a goal in itself.

Let's merge!